### PR TITLE
FF8: Various enhancements and fixes

### PR DESCRIPTION
**Pull request by myst6re**
## Summary

- Fix gamepad icons not displayed
- Use libPNG to decode PNGs, instead of bimg
- Use DirectXTex to decode DDS and decompress only the relevant lod for faster loading
- Minimize memory allocations with external textures to allow bigger textures and prevent game crashes (the game is limited to 2GB if not patched, and 4GB if patched, due to its x86 nature)

### Motivation

Fix things

### ACKs

- [X] I have updated the [Changelog.md](https://github.com/julianxhokaxhiu/FFNx/blob/master/Changelog.md) file
- [ ] I did test my code on FF7
- [X] I did test my code on FF8

### CODE REVIEW

File: vcpkg.json [hunk 1]
```diff
@@ -35,6 +35,7 @@
     },
     "imgui",
     "libpng",
+    "directxtex",
```
Comment (julianxhokaxhiu) [inline vcpkg.json:38]: Please add the version you target on the `overrides` section. Ty

**Comment by myst6re** [comment]
Ready! I made 4 different commits, one for each subject, you can read them separatly to make the review simpler.

File: CMakePresets.json [hunk 2]
```diff
@@ -19,7 +19,7 @@
         "strategy": "set"
       },
       "environment": {
-        "VCPKG_ROOT": "C:/vcpkg",
+        "VCPKG_ROOT": "E:/Programs/vcpkg",
```
Comment (julianxhokaxhiu) [inline CMakePresets.json:22]: Can you please revert this change?

**Review by julianxhokaxhiu — APPROVED** [review]

**Comment by julianxhokaxhiu** [comment]
Thank you!
